framework/src/modules/chain/transactions/votes_weight.js

```diff
@@ -224,9 +224,13 @@ const prepare = async (stateStore, transaction) => {
 	];
 
 	const cacheFilter = senderRecipientVotedPublicKeys.map(publicKey => ({
-		address: getAddressFromPublicKey(publicKey),
+		publicKey,
```

Comment:
I think we should use consistently primary key search otherwise we will always need more index